packages/blockchain-wallet-v4-frontend/src/components/Flyout/index.tsx

```diff
@@ -73,8 +73,7 @@ const FlyoutModal = styled(AnimatedModal)`
   color: ${(props) => props.theme.grey700};
   ${media.mobile`
     width: 100%;
-    height: calc(100vh - 80px);
-    padding-bottom: 20px;
+    height: 100vh;
```

Comment:
On production iOS Mobile Safari there is a banner that I believe is what that 80px is accounting for. I think this change could possibly push the flyout down an additional 80px more than it should and could cover up the buy button. You may want to use something other than `vh` in this case.
![image](https://user-images.githubusercontent.com/57680122/154363736-b83a41e9-34d2-4f4d-8729-be684083557b.png)

Reply:
I was not able to open the page with my device
should someone with an iPhone try to debug this?